generic/htx_test.py

```diff
@@ -21,8 +21,10 @@
 import os
 import time
 import shutil
+import re
+
 from avocado import Test
-from avocado.utils.software_manager.manager import SoftwareManager
+from avocado.utils.software_manager import SoftwareManager
```

Comment:
Please clone the latest misc test and then do the modification.
.manager was recent change and should not be removed.